dev/parm/config/gcafs/config.resources

```diff
@@ -554,6 +554,28 @@ case ${step} in
     memory="48GB"
     ;;
 
+  "marinebmatinit")
+    npes=16
```

Comment:
Not necessary to address: if `marinebmatinit` only copies files/directories, I think it could run on just 1 task, similar e.g. to `marineanlinit`. That is unless something can be done in parallel in this task, I'm not sure.

Reply:
I figured this would be the case, though I just copied the entry from `marinebmat`.

Reply:
changed